### PR TITLE
Interference when trying to put a bluespace item into another

### PR DESCRIPTION
**Pull request by hyperioo**
## About The Pull Request

When you try to put a bluespace item (backpack, trashbag, pouch, belt, satchel) into another one, your arm explodes.

Add a new item flag "BLUESPACE"

## Why It's Good For The Game

Having your arm blown off is more fun than just one of the two items disappearing. Trying to cheat physics should have some consequences.

## Changelog
:cl: Hyperio
add: Interference when trying to put a bluespace item into another
code: New BLUESPACE item flag
/:cl:

### CODE REVIEW

**Comment by hyperioo** [comment]
Closing and reopening for travis.

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -84,10 +84,23 @@
...
 
+/obj/item/weapon/storage/backpack/holding/New()
+	..()
+	item_flags |= BLUESPACE
+
 /obj/item/weapon/storage/backpack/holding/attackby(obj/item/weapon/W as obj, mob/user as mob)
-	if(istype(W, /obj/item/weapon/storage/backpack/holding))
-		to_chat(user, SPAN_WARNING("The Bluespace interfaces of the two devices conflict and malfunction."))
-		qdel(W)
+	if(W.item_flags & BLUESPACE)
```
Comment (TheShown911) [inline code/game/objects/items/weapons/storage/backpack.dm:92]: A better way to do all this is to put this code under /obj/item/weapon/storage/ and check if both the item being stored and `src` have the bluespace flag.

Comment (TorinoFermic) [reply code/game/objects/items/weapons/storage/backpack.dm:92]: Indeed, there's awfully many copy and paste in this PR

**Review by drexample — CHANGES_REQUESTED** [review]
@hyperioo What other maintainers requested + resolve conflicts

**Review by genessee596 — APPROVED** [review]
Delegate stamp of approval.

Concept-wise, this is more fun than just deleting one of the items, while not being as abusable as creating a singularity like on other codebases.

Needs meet maintainer requests and receive an approval from a maintainer before it becomes mergable, though.

**Comment by genessee596** [comment]
This PR is getting stale.

If you don't make a response letting me know that you're still working on this, the PR will be closed in one day.

Thanks.

**Comment by TorinoFermic** [comment]
Closing the PR for lack of reply

**Pull request closed without merging**